src/Common/CodeEditor/types.ts

```diff
@@ -84,6 +84,7 @@ export enum CodeEditorThemesKeys {
     deleteDraft = 'delete-draft',
     unpublished = 'unpublished',
     vs = 'vs',
+    networkStatusInterface = 'network-status-interface',
```

Comment:
Question: Shouldn't this be dark instead?

Reply:
Actually for dark mode the colour palette would be different